dsp/dsp.cpp

```diff
@@ -8,494 +8,6 @@
 #include <unordered_set>
 
 #include "dsp.h"
-#include "json.hpp"
```

Comment:
This is moved to `NAM/dsp.cpp` with no modification? Please confirm.

Reply:
Like dsp.h - dsp.cpp gets split between dsp/dsp.cpp and NAM/dsp.cpp. The former has the noise gate, IR and EQ base class, and the latter has the network model stuff.

It also removes IPlug dependencies by using "double" instead of "iplug::sample".